packages/design-system/src/components/N8nNotice/Notice.vue

```diff
@@ -77,24 +77,34 @@ export default Vue.extend({
 				},
 			);
 		},
-		onClick(e) {
-			if (e.target.localName !== 'a') return;
+		onClick(event: NoticeClickEvent) {
```

Comment:
is not this basically `MouseEvent`? target is HTMLElement..

Reply:
Tried this but `event.target` defaults to `EventTarget` so had to intersect it.

Reply:
Never mind, found the solution.